ClientLibrary/Snippets/Microsoft.TeamServices.Samples.Client/Wiki/WikiPagesSample.cs

```diff
@@ -0,0 +1,285 @@
+﻿using Microsoft.TeamFoundation.SourceControl.WebApi;
+using Microsoft.TeamFoundation.Wiki.WebApi;
+using Microsoft.VisualStudio.Services.WebApi;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Microsoft.TeamServices.Samples.Client.Wiki
+{
+    [ClientSample(WikiConstants.AreaName, WikiConstants.PagesResourceName)]
+    public class WikiPagesSample : ClientSample
+    {
+        [ClientSampleMethod]
+        public WikiPageResponse CreateWikiPage()
+        {
+            VssConnection connection = this.Context.Connection;
+            WikiHttpClient wikiClient = connection.GetClient<WikiHttpClient>();
+
+            Guid projectId = ClientSampleHelpers.FindAnyProject(this.Context).Id;
+
+            List<WikiV2> wikis = wikiClient.GetAllWikisAsync(projectId).SyncResult();
+
+            if (wikis == null || wikis.Count == 0)
+            {
+                Console.WriteLine("No wikis exist to create wiki page");
+
+                return null;
+            }
+
+            WikiV2 wiki = wikis[0];
+            WikiPageCreateOrUpdateParameters parameters = new WikiPageCreateOrUpdateParameters()
+            {
+                Content = "Wiki page content"
+            };
+
+            WikiPageResponse wikiPageResponse = wikiClient.CreateOrUpdatePageAsync(
+                parameters,
+                project: wiki.ProjectId,
+                wikiIdentifier: wiki.Id,
+                path: "SamplePage" + new Random().Next(1, 999),
+                Version: null).SyncResult();
+
+            Console.WriteLine("Create page '{0}' in wiki '{1}'", wikiPageResponse.Page.Path, wiki.Name);
+
+            return wikiPageResponse;
+        }
+
+        [ClientSampleMethod]
+        public WikiPageResponse GetWikiPageJson()
```

Comment:
I would avoid saying "JSON" since nothing about this sample deals with JSON. I think you want to say "Metadata" instead (e.g. `GetWikiPageMetadata`).